docs/matrices.rst

```diff
@@ -21,6 +21,7 @@ Matrices
     toqito.matrices.pauli
     toqito.matrices.shift
     toqito.matrices.standard_basis
+    toqito.matrices.cyclic_permutation
```

Comment:
I know this is a minor nit-pick, but would you mind placing this alphabetically in the list? Thank you!